ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java

```diff
@@ -2087,62 +2084,16 @@ public void getBlobsOnTape() throws Exception {
-                "| coffeehouse/jk/ColumbinesGrow.mp3               | coffeehouse |   5050747 | c85fc175-116a-4bcf-a77a-5ea240a5de3a |\n" +
-                "| coffeehouse/jk/Columbines_295x166.jpg           | coffeehouse |     10528 | b70bd4ab-90d2-41fd-83d2-572fb3d1c8ca |\n" +
-                "| coffeehouse/jk/Misty_2015.m4a                   | coffeehouse |  10396369 | e4769cd2-3aa6-4628-887c-ad51768656c5 |\n" +
-                "| coffeehouse/jk/RedRiverValley.m4a               | coffeehouse |  77080710 | 9ffa7e9c-6939-4808-996e-e42fcf8bacb5 |\n" +
-                "| coffeehouse/jk/RedRiverValley.mp3               | coffeehouse |   6363965 | 564a1bc1-33a0-41f3-af28-fbf79f331d0e |\n" +
-                "| coffeehouse/jk/UnforgetWonderful_295x166.jpg    | coffeehouse |     10724 | b2671db7-1a4a-4577-8419-f17ead63d321 |\n" +
-                "| coffeehouse/jk/Unforgettable-WonderfulWorld.m4a | coffeehouse | 110054089 | 71807ee9-2db9-4145-b01d-3d2aaae37061 |\n" +
-                "| coffeehouse/jk/Unforgettable-WonderfulWorld.mp3 | coffeehouse |   7520930 | e50d5fc8-8fbf-4206-b495-05bb8be539ec |\n" +
-                "| coffeehouse/jk/WhereOrWhen.m4a                  | coffeehouse |  51272203 | 9156aab6-88fa-49b0-a0e1-c230d247957e |\n" +
-                "| coffeehouse/jk/WhereOrWhen.mp3                  | coffeehouse |   5647581 | 0f5541b9-8c4d-4ed8-bd1d-9e62173bdf4a |\n" +
-                "| coffeehouse/jk/WhereOrWhen_295x166.jpg          | coffeehouse |     11263 | 03b2e1c7-f80c-437a-912d-b09015dba484 |\n" +
-                "| coffeehouse/jk/im_in_the_mood.m4a               | coffeehouse |  11207247 | 667d94f6-b341-45f7-bd91-706af52d8e77 |\n" +
-                "| coffeehouse/jk/im_in_the_mood_200.jpg           | coffeehouse |      8621 | f7f65e20-4ea2-4629-9c22-ddf9cbc76b99 |\n" +
-                "| coffeehouse/witchcraft.mp3                      | coffeehouse |   6409093 | 92a40cff-63a6-4520-81a9-80afa03a1973 |\n" +
-                "+-------------------------------------------------+-------------+-----------+--------------------------------------+\n";
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n" +
+                "|        Name        |    Bucket   | Size |                  Id                  |              Version-Id              |\n" +
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n" +
+                "| 123456789.txt      | coffeehouse |    9 | 53452a07-699a-4c27-8de5-95aa0a431df1 | 53452a07-699a-4c27-8de5-95aa0a431df1 |\n" +
+                "| Always_295x166.jpg | coffeehouse | 9172 | 7989ad4a-47a5-41ac-8814-3746e4e20679 | 7989ad4a-47a5-41ac-8814-3746e4e20679 |\n" +
+                "+--------------------+-------------+------+--------------------------------------+--------------------------------------+\n";
```

Comment:
why does the output is so short now? i don't see you change the number files of the test

Reply:
I removed the extra entries from the test to simplify modifying it.  I changed the xml input, and the table formatted response.